app/server/models/User.js

```diff
@@ -6,97 +6,129 @@ var mongoose   = require('mongoose'),
 
 var profile = {
 
-  // Basic info
-  name: {
+  // Personal Info
+  firstName: {
     type: String,
     min: 1,
     max: 100,
   },
 
+  lastName: {
+    type: String,
+    min: 1,
+    max: 100,
+  },
+
+  phoneNumber: String,
+
+  email: {
+    type: String,
+    min: 1,
+    max: 100,
+  },
+
+  gender: {
+    type: String,
+    enum : {
+      values: 'M F N'.split(' ')
+    }
+  },
+
   adult: {
     type: Boolean,
     required: true,
     default: false,
   },
 
+
+  shirtSize: {
+    type: String,
+    enum: {
+      values: 'XS S M L XL XXL'.split(' ')
+    }
+  },
+
+  emergencyContact: {
+    name: String,
+    workNumber: String,
+    cellNumber: String,
+    relationship: {
+      type: String,
+      min: 1,
+      max: 100, 
+    },
+  },
+
+//School Information
   school: {
     type: String,
     min: 1,
     max: 150,
   },
 
-  graduationYear: {
+  schoolYear: {
     type: String,
     enum: {
-      values: '2016 2017 2018 2019'.split(' '),
+      values: 'High-School Freshman Sophomore Junior Senior'.split(' '),
     }
   },
 
-  description: {
+  major: String,
+
+//Additional Logistics
+
+  dietaryRestrictions: {
     type: String,
-    min: 0,
-    max: 300
+    enum: {
+      values: 'Halal Kosher Vegan Vegetarian Gluten-Free'.split(' '),
+    }
   },
 
-  essay: {
+  needsReimbursement: Boolean,
+
+  reimbursementYes: {
     type: String,
     min: 0,
-    max: 1500
+    max: 150
   },
 
-  // Optional info for demographics
-  gender: {
+//Bitcamp
+
+  amtHackathons: {
     type: String,
-    enum : {
-      values: 'M F O N'.split(' ')
+    enum: {
+      values: '0 1-5 6-10 11-15 16-20 20+'.split(' '),
     }
   },
+  whyBitcamp: {
+    type: String,
+    min: 0,
+    max: 250
```

Comment:
I think these are character counts, not word counts. We probably want to make it bigger

Reply:
done. upped to 2500